### DISCUSSION TITLE
Humidity with Alexa

### DISCUSSION THREAD

**Discussion by Schwenkner** [t0bst4r/home-assistant-matter-hub#329] [Q&A]
I have asked in HA forum if humidity sensor is already working as the documentation states. And it was agreed to work with Google Home.
I'm using Alexa and I cannot ask for humidity. The sensor is shown in Alexa app and displays the correct value. But is asking for the value I get "This feature is currently not supported". Asking for something like pressure I get "Not available". Thus I assume "not supported" means as it states, supported but not available. Is there still a bug or something necessary to setup in the entity? With Google Home the user confirmed that it is working.

**Comment by danielbrunt57** [comment]
> The sensor is shown in Alexa app

What is the sensor?

**Reply by Schwenkner** [reply]
I'm discussing this also here:
[https://github.com/t0bst4r/home-assistant-matter-hub/issues/331](url)
